src/tests/e2etest/CommandRunnerTests.cs

```diff
@@ -64,67 +60,53 @@ public static CommandArguments CreateCommand(string newCommandId, string newArgu
         {
             return new CommandArguments
             {
-                CommandId = newCommandId,
-                Arguments = newArguments,
-                Action = newAction,
-                Timeout = newTimeout,
-                SingleLineTextResult = newSingleLinetextResult
+                commandId = newCommandId,
```

Comment:
Is there a way to keep these to be `PascalCased`? There was a change a while back that fixed this: #181

Reply:
oups, yes, i saw that. I didn't intend to have those changes in there. I will change it back.